.github/workflows/nightly-linux.yml

```diff
@@ -24,6 +24,28 @@ jobs:
         toolchain: ['Clang']
         clang_plugins: [false]
 
+        include:
+          - os_name: 'Linux'
+            os: ubuntu-24.04
+            arch: 'x86_64'
+            build_preset: 'Distribution_CI'
+            toolchain: 'GNU'
+            clang_plugins: false
+
+          - os_name: 'macOS'
+            os: macos-15
+            arch: 'arm64'
+            build_preset: 'Distribution_CI'
+            toolchain: 'Clang'
+            clang_plugins: false
```

Comment:
`nightly-linux.yml` is running macOS builds? Maybe call this `nightly-distribution.yml`?

Reply:
it also runs a sanitizer build on arm64 linux, so nightly-distribution isn't quite the right name either 😅

Reply:
Went with `nightly-lagom.yml`, since these are nightly jobs that invoke `lagom-template.yml`